DXRFixes/DeusEx/Classes/Carcass.uc

```diff
@@ -281,6 +283,13 @@ function bool TryLootItem(DeusExPlayer player, Inventory item)
         return true;
     }
 
+    if (action == 2 && Human(player).ConsumableWouldHelp(item) && DeusExPickup(item) != None) { // consume
```

Comment:
should auto consume items that won't help be dropped or looted?

Reply:
Yeah, I was unsure about this. My thinking was to ultimately defer to default behavior. But, I don't know, maybe the only place it would actually matter really is in speedruns, in which case you'd want them dropped?